src/Core/src/Platform/Tizen/WindowExtensions.cs

```diff
@@ -113,5 +113,19 @@ static void OnBackButtonPressed(Window platformWindow)
 				s_windowCloseRequestHandler[platformWindow].Invoke();
 		}
 
+		internal static void UpdateX(this Window platformWindow, IWindow window) =>
+			platformWindow.UpdateUnsupportedCoordinate(window);
+
+		internal static void UpdateY(this Window platformWindow, IWindow window) =>
+			platformWindow.UpdateUnsupportedCoordinate(window);
+
+		internal static void UpdateWidth(this Window platformWindow, IWindow window) =>
+			platformWindow.UpdateUnsupportedCoordinate(window);
+
+		internal static void UpdateHeight(this Window platformWindow, IWindow window) =>
+			platformWindow.UpdateUnsupportedCoordinate(window);
+
+		internal static void UpdateUnsupportedCoordinate(this Window platformWindow, IWindow window) =>
+			window.FrameChanged(platformWindow.WindowPositionSize.ToDP());
```

Comment:
No problems with current tizen implementation at the moment. We need more investigation for supporing multi window and window resizing on tizen (with NUI), hopefully it should be supported. If we find a way, I will submit separate PR the actual tizen implementation. 🤙